tests/testthat/_snaps/rename_ard_columns.md

```diff
@@ -10,13 +10,13 @@
        1 Placebo    65-80  catego~ n         n          42     42       <int>  <NULL> 
        2 Placebo    65-80  catego~ N         N          86     86       <int>  <NULL> 
        3 Placebo    65-80  catego~ p         %           0.488 48.8     <fn>   <NULL> 
-       4 Xanomelin~ 65-80  catego~ n         n          55     55       <int>  <NULL> 
-       5 Xanomelin~ 65-80  catego~ N         N          84     84       <int>  <NULL> 
-       6 Xanomelin~ 65-80  catego~ p         %           0.655 65.5     <fn>   <NULL> 
-       7 Xanomelin~ 65-80  catego~ n         n          47     47       <int>  <NULL> 
-       8 Xanomelin~ 65-80  catego~ N         N          84     84       <int>  <NULL> 
-       9 Xanomelin~ 65-80  catego~ p         %           0.560 56.0     <fn>   <NULL> 
-      10 Placebo    <65    catego~ n         n          14     14       <int>  <NULL> 
+       4 Placebo    <65    catego~ n         n          14     14       <int>  <NULL> 
+       5 Placebo    <65    catego~ N         N          86     86       <int>  <NULL> 
+       6 Placebo    <65    catego~ p         %           0.163 16.3     <fn>   <NULL> 
+       7 Placebo    >80    catego~ n         n          30     30       <int>  <NULL> 
+       8 Placebo    >80    catego~ N         N          86     86       <int>  <NULL> 
+       9 Placebo    >80    catego~ p         %           0.349 34.9     <fn>   <NULL> 
+      10 Xanomelin~ 65-80  catego~ n         n          55     55       <int>  <NULL> 
       # i 17 more rows
```

Comment:
These snapshots that have a truncated prints make me a little nervous that there is something creeping in the unprinted rows/columns. What do you think?

Was this snapshot meant to test the column order only? if so, do you think we should have originally done `expect_equal()` on the column names?

Reply:
@ddsjoberg Thanks! I have added the data frame conversion for full visibility.

As for your second question, this test addresses the unnesting option as well so I think we want the full print out/